src/components/TextAreaInput/TextAreaInput.tsx

```diff
@@ -15,13 +14,14 @@ const TextAreaInput: React.FC<TextAreaInputProps> = ({ name, placeholder }) => {
   const errorMessage = errors[name]?.message as string | undefined;
 
   return (
-    <div className="text-area">
+    <div className="flex flex-col items-center">
       <textarea
-        className="text-area-input"
+        className="w-full h-[167px] rounded-[12px] p-3.5 bg-[#E9E8D5] mb-2.5 box-border 
```

Comment:
You need to make the height dependent on which form input is in question as opposed to hard coding it. 

You also need to use the background color from the config file as opposed to using hex.